src/main/resources/reference.conf

```diff
@@ -20,4 +20,6 @@ gargoyle {
         defaultTokenSessionHours = 8
         maxTokenSessionHours = 24
     }
+
+    npa: []
```

Comment:
should we add the same to applicaiton.conf?

Reply:
Since we don't want to have NPAs hardcoded in our codebase, I would say no. When you deploy our current STS service, you have to make sure yourself that you set any NPAs you want to have in the application.conf